### ISSUE TITLE
Problem on Training: No model found in config file

### ISSUE THREAD

**Issue by sst19910323** [qqwweee/keras-yolo3#54]
Hi there,

My datasat has one class, format is as VOC2007
Annotation file looks like that.
J:\VOC2007\JPEGImages\000036.jpg 49,176,67,191,0 411,366,427,383,0 462,154,481,169,0

I can generate ep030*****.h5 and the loss value looks right, but they are smaller than the .h5 file I converted from yolov3.weight.

When I try to run yolo.py with my own model, it response "No model found in config file."

**Comment by qqwweee** [comment]
`ep**.h5` only contains weights. You can run
`model = yolo_body(Input(shape=(None, None, 3)), 3, num_classes)
model.load_weights('ep**.h5')`
to use it. Or `model.save('model.h5')` to get the model file.

Or in `train.py`, `save_weights_only=False` in `ModelCheckpoint` to make `ep**.h5` a model file instead of a weigths file.

**Comment by cad-ml** [comment]
I have the same problem. Have you found the answer?

**Comment by ElegantLin** [comment]
Hi, I solved this problem according to https://blog.csdn.net/weixin_38136156/article/details/86437715. Hope to help you.
